platform/machine/qemu/cluster.go

```diff
@@ -132,7 +132,15 @@ ExecStartPost=/usr/bin/ln -fs /run/metadata/flatcar /run/metadata/coreos
 
 	// This uses path arguments with path values being
 	// relative to the folder created for this machine
-	qmCmd, extraFiles, err := platform.CreateQEMUCommand(qc.flight.opts.Board, qm.id, qc.flight.opts.BIOSImage, qm.consolePath, confPath, qc.flight.diskImagePath, conf.IsIgnition(), options)
+	biosImage := qc.flight.opts.BIOSImage
+	if !strings.HasPrefix(biosImage, "/") {
+		cwd, err := os.Getwd()
+		if err != nil {
+			return nil, fmt.Errorf("getting cwd: %v", err)
+		}
+		biosImage = filepath.Join(cwd, biosImage)
+	}
```

Comment:
https://pkg.go.dev/path/filepath#Abs
```suggestion
	biosImage, err := filepath.Abs(qc.flight.opts.BIOSImage)
        if err != nil {
                return nil, fmt.Errorf("failed to canonicalize bios path: %v", err)
        }
```

Reply:
Thanks